src/Link.ts

```diff
@@ -40,7 +56,7 @@ export class Link extends WidgetBase<LinkProperties> {
 
 	protected render(): DNode {
 		const { to } = this.properties;
-		const props = { ...this.properties, to: undefined, isOutlet: undefined, params: undefined, routerKey: undefined };
+		const props = { ...this.properties, onclick: this._onClick, onClick: undefined, to: undefined, isOutlet: undefined, params: undefined, routerKey: undefined, router: undefined };
```

Comment:
This pattern is sort of getting messy and potentially error prone... I don't have any suggestions, but it feels like this could be abstracted into a more clear function...